### DISCUSSION TITLE
Does anyone know why AUC value from create_model() and from plot_models are different?

### DISCUSSION THREAD

**Discussion by gh-ga** [pycaret/pycaret#3256] [General]
The mean AUC value from create_model is 0.8305 while the AUC value in the plot_model function is 0.77. Does anyone know the reason for the discrepancy? 

I've seen this issue in the work of others (i.e. in online tutorials - compare 9.3 with 10.1 in [this tutorial](https://www.pycaret.org/tutorials/html/CLF101.html)) 
![rf AUC and accuracy - final model_1 14 23](https://user-images.githubusercontent.com/65797661/212457659-45404325-1b62-407e-999b-fb75de9499b9.png)
![plot_model for rf_1 14 23](https://user-images.githubusercontent.com/65797661/212457660-83691539-d9bb-439f-a6a8-d06b3eed3289.png)

**Comment by U-C-J** [comment]
Create model AUC is for internal cross validation. plot model auc is for test set. That is why they are different.
